lib/PuppeteerSharp/BrowserFetcher.cs

```diff
@@ -114,7 +114,7 @@ public async Task<bool> CanDownloadAsync(int revision)
                 var client = WebRequest.Create(url);
                 client.Proxy = _webClient.Proxy;
                 client.Method = "HEAD";
-                using (var response = await client.GetResponseAsync().ConfigureAwait(false) as HttpWebResponse)
+                using (var response = (HttpWebResponse)await client.GetResponseAsync().ConfigureAwait(false))
```

Comment:
no null check after an `as` implied it can only be that type. hence a cast is better, since if the assumption is incorrect you will get a useful cast exception instead of a useless null ref exception

Reply:
Did you go one by one or using a tool?
It would be cool to find an analyzer checking this.